front/gatsby/src/components/Export.jsx

```diff
@@ -18,44 +18,22 @@ const mapStateToProps = ({ applicationConfig }) => {
   return { applicationConfig }
 }
 
-const Export = (props) => {
-  const processEndpoint = props.applicationConfig.processEndpoint
-  const exportEndpoint = props.applicationConfig.exportEndpoint
+const Export = ({ book, bookId, name, title, version, revision, versionId, applicationConfig }) => {
+  const { processEndpoint, exportEndpoint } = applicationConfig.processEndpoint
```

Comment:
Tu extrais 2 valeurs depuis une même chaîne ?

Reply:
Ooops non c'est une erreur:

```suggestion
  const { processEndpoint, exportEndpoint } = applicationConfig
```